### PR TITLE
feat(route): add /fortnite/news

### LINKED ISSUE #10868: 游戏 《堡垒之夜》（ Fortnite） 的 新闻
### 类型

其他

### 网站地址

https://www.epicgames.com/fortnite/en-US/news

### 网站描述

这是《堡垒之夜》官方网站新闻发布的页面，可以通过这个页面了解该游戏的最新消息！

### 需要生成什么内容？

- 每个新闻的标题
- 发布日期，并做好排序
- 新闻的链接
- 如果有余力，希望能在每个条目的description里生成每个新闻的内容，就是点击后出现的那些内容。

### 额外描述

《堡垒之夜》真的太好玩了！希望路过的高人们能有空帮忙制作一下，多谢啦！ ：）

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by lyqluis**
## 该 PR 相关 Issue / Involved Issue

Close #10868 

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/fortnite/news
/fortnite/news/lang=ja
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [x] `Puppeteer`

### CODE REVIEW

**Comment by ruleroller** [comment]
感谢有缘人大佬！好用。

File: lib/v2/fortnite/maintainer.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/fortnite/news/:options?': ['lyqluis'],
```
Comment (TonyRL) [inline lib/v2/fortnite/maintainer.js:2]:
```suggestion
    '/news/:options?': ['lyqluis'],
```

File: lib/v2/fortnite/news.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    await page.close();
+    browser.close();
+
+    const { blogList: list } = data;
+    const items = await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.link, () => ({
+                title: item.title,
+                link: `${rootUrl}/${path}/${item.slug}?lang=${language}`,
+                pubDate: timezone(parseDate(item.date)),
```
Comment (TonyRL) [inline lib/v2/fortnite/news.js:47]:
```suggestion
                pubDate: parseDate(item.date),
```

File: docs/game.md [hunk 3]
```diff
@@ -103,6 +103,20 @@ pageClass: routes
 
 <Route author="nczitzk" example="/fgo/news" path="/fgo/news"/>
 
+## Fortnite
+
+### news
```
Comment (TonyRL) [inline docs/game.md:108]:
```suggestion
### News
```

File: lib/v2/fortnite/radar.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    'fortnite.com': {
+        _name: 'Fortnite',
+        '.': [
+            {
+                title: 'news',
```
Comment (TonyRL) [inline lib/v2/fortnite/radar.js:6]:
```suggestion
                title: 'News',
```

File: docs/game.md [hunk 5]
```diff
@@ -103,6 +103,20 @@ pageClass: routes
 
 <Route author="nczitzk" example="/fgo/news" path="/fgo/news"/>
 
+## Fortnite
```
Comment (TonyRL) [inline docs/game.md:106]: Please add en docs as well

File: docs/game.md [hunk 6]
```diff
@@ -103,6 +103,20 @@ pageClass: routes
...
+
+### news
+
+<Route author="lyqluis" example="/fortnite/news" path="/fortnite/news/:options?" :paramsDesc="['参数']" radar="1">
```
Comment (TonyRL) [inline docs/game.md:110]:
```suggestion
<Route author="lyqluis" example="/fortnite/news" path="/fortnite/news/:options?" :paramsDesc="['参数']" radar="1" puppeteer="1">
```

File: docs/game.md [hunk 7]
```diff
@@ -103,6 +103,20 @@ pageClass: routes
...
+
+-   `options.lang`，可选，语言，实例：`/fortnite/news/lang=en-US`，常见语言见下表，更多语言参考[官网](https://www.fortnite.com/news)
```
Comment (TonyRL) [inline docs/game.md:112]:
```suggestion
-   `options.lang`，可选，语言，实例：`/fortnite/news/lang=en-US`，常见语言见下表，更多语言参考 [官网](https://www.fortnite.com/news)
```

**Pull request merged**